### ISSUE TITLE
[Enhancement] Support for correct Message item with disclosure

### ISSUE THREAD

**Issue by dennispedersen-bd** [kirbydesign/designsystem#4309]
- [ ] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement

Support for the correct Message item with disclosure in Kirby. Both cookbook example and Figma library.

## Describe the solution you'd like

Figma:
- Add new title variant called "Title - Message Item Disclosure" that contains both: Title, time stamp and disclosure. ✅
- Publish new variant in Figma Library

Cookbook:
- Update the recipe example in cookbook with direction="horizontal" and rename from complex structure to "Message Item Recipe"
- The recipe should show time detail and custom disclosure icon with correct spacing
- - Custom placement of icon
- - Custom placement of detail
- Enable wrap with truncation after af chosen line amount. The detail example should truncate after to lines.
- Chek baseline alignment of Title and date with different text sizes

Design: 

https://www.figma.com/design/KRnTge4tZYtAqY2LD9uuC6/Text-resize?node-id=2682-4064&t=XGXSQC1rZZFFxqBa-1

Figma Component library branch: 
https://www.figma.com/design/6Fgk0gzWGxJYPYNS0RdDgX/branch/xg2qQZ7On9T4PqcZXW2L8K/01-Kirby-Components?m=auto&node-id=1069-1494&t=2KfINMSIsl6uikBR-1

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#ux-refinement)
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](../CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".
